dotnet/src/SemanticKernel/SemanticKernel.csproj

```diff
@@ -6,7 +6,7 @@
   <Import Project="$(RepoRoot)/dotnet/nuget/nuget-package.props" />
 
   <PropertyGroup>
-    <AssemblyName>Microsoft.SemanticKernel</AssemblyName>
+    <AssemblyName>Microsoft.SemanticKernel.Core</AssemblyName>
```

Comment:
I think all of this should move to SemanticKernel.Core/ folder and rename the csproj too. These should match the nugets.

Reply:
can we move the files separately, to keep the PR small?

Reply:
yup, sounds good.